server/aws-lsp-codewhisperer/src/language-server/chat/contexts/documentFqnExtractor.ts

```diff
@@ -0,0 +1,161 @@
+import { DocumentSymbol, SymbolType } from '@amzn/codewhisperer-streaming'
+import { FqnExtractorOutput, FullyQualifiedName, fqnExtractor } from '@aws/lsp-fqn'
+import { Range, TextDocument } from 'vscode-languageserver-textdocument'
+import { Features, Result } from '../../types'
+
+export interface DocumentFqnExtractorConfig {
+    nameMinLength?: number
+    nameMaxLength?: number
+    maxSymbols?: number
+    timeout?: number
+    logger?: Features['logging']
+}
+
+export type FqnSupportedLanagues =
```

Comment:
typo

Reply:
Cann't those be defined in 1 place, e.g. `fqn` package?

Reply:
Yes, it should be. I will move them, thanks!